pkg-config: needs pthread as private library

libtirpc needs pthreads. This is automatically pulled in when linking
with the shared library, thanks to the DT_NEEDED ELF tags.

But for a static library, there is no such mechanism to identify
required libraries.

Thus, add -pthread to the private libraries in the pkg-config file.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

---
Patch sent upstream:
    http://sourceforge.net/p/libtirpc/mailman/message/33072076/

diff -durN libtirpc-0.2.4.orig/libtirpc.pc.in libtirpc-0.2.4/libtirpc.pc.in
--- libtirpc-0.2.4.orig/libtirpc.pc.in	2013-12-09 21:59:51.000000000 +0100
+++ libtirpc-0.2.4/libtirpc.pc.in	2014-11-23 14:02:58.677760108 +0100
@@ -8,4 +8,5 @@
 Requires:
 Version: @PACKAGE_VERSION@
 Libs: -L@libdir@ -ltirpc
+Libs.private: -pthread
 Cflags: -I@includedir@/tirpc
